x/pairing/keeper/grpc_query_provider_monthly_payout.go

```diff
@@ -57,6 +57,15 @@ func (k Keeper) ProviderMonthlyPayout(goCtx context.Context, req *types.QueryPro
 			}
 			totalTokenAmount := plan.Price.Amount
 			totalCuTracked := subObj.MonthCuTotal - subObj.MonthCuLeft
+			// Sanity check - totalCuTracked > 0
+			if totalCuTracked <= 0 {
+				return nil, utils.LavaFormatWarning("critical: Attempt to divide by zero or negative number", nil,
```

Comment:
same comment as before

Reply:
Fixed in 20bf6f5ad2b263ea30b7641558c140ceca7a9293